rebasehelper/tests/test_specfile.py

```diff
@@ -438,14 +438,14 @@ def test_typo_fix_spec_hook(self, spec_object):
         assert TypoFixHook.get_name() in spec_hooks_runner.spec_hooks
         assert '- This is chnagelog entry with some indentional typos\n' in spec_object.spec_content
         spec_object.category = 'sample'
-        spec_hooks_runner.run_spec_hooks(spec_object, spec_object)
+        spec_hooks_runner.run_spec_hooks(spec_object, spec_object, **{'disable_spec_hook': []})
```

Comment:
Why don't you simply use `disable_spec_hook=[]`?

Anyway, I think it would make sense not to pass empty list here, but default to empty list in `run_spec_hooks()` in case `disable_spec_hook` kwarg doesn't exist.